sources/api/schnauzer/src/helpers/mod.rs

```diff
@@ -1314,6 +1320,21 @@ fn oci_spec_resource_limits(
     Ok(serde_json::from_value(value.clone())?)
 }
 
+fn adjust_resource_limits_for_efa(
+    rlimits: &mut HashMap<OciDefaultsResourceLimitType, OciDefaultsResourceLimitV1>,
+) {
+    if !rlimits.contains_key(&OciDefaultsResourceLimitType::MaxLockedMemory) {
+        let max_locked_memory_limits = OciDefaultsResourceLimitV1 {
+            soft_limit: -1,
+            hard_limit: -1,
+        };
+        rlimits.insert(
+            OciDefaultsResourceLimitType::MaxLockedMemory,
+            max_locked_memory_limits,
+        );
+    }
```

Comment:
Clippy is not happy with this. Will change with  
```suggestion
    rlimits.entry(OciDefaultsResourceLimitType::MaxLockedMemory).or_insert(OciDefaultsResourceLimitV1 {
        soft_limit: -1,
        hard_limit: -1,
    });
```

Will give it a last test before pushing.